capella_model_explorer/__main__.py

```diff
@@ -147,10 +139,10 @@ def run_local_dev() -> None:
         "*.py",
         "capella_model_explorer.app:app",
     ]
-    print(" ".join(uvicorn_cmd))
-    tailwind_proc = build_css(watch=True)
-    assert tailwind_proc is not None
-    time.sleep(1)  # avoid direct uvicorn reload when css file is written
+    logger.info(" ".join(uvicorn_cmd))
+    assert (tailwind_proc := build_css(watch=True))
+    time.sleep(1)
+
```

Comment:
My aim is to avoid unnecessary empty lines. I am not there yet and at the same time I wish that we avoid them.

Reply:
I added the empty line to better the readability.